crates/modelardb_common/src/test/mod.rs

```diff
@@ -133,3 +138,19 @@ pub fn compressed_segments_record_batch_with_time(
     )
     .unwrap()
 }
+
+/// Return invalid but lazy connections to a PostgreSQL [`TableMetadataManager`] and a
```

Comment:
The connections themselves are invalid since if you try to actually connect to them they fail. I am not sure if there is a better word to describe this.

Reply:
ok